.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
 
       - name: Get all official build targets
         id: get-targets
-        run: echo "::set-output name=targets::$(make targets-ci-print | jq -R -c 'split(" ")')"
+        run: echo "{targets}=$(make targets-ci-print | jq -R -c 'split(" ")')" >> $GITHUB_OUTPUT
```

Comment:
Seeing this link: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter it seems the `{}` are not necessary, so maybe...
```suggestion
        run: echo "targets=$(make targets-ci-print | jq -R -c 'split(" ")')" >> $GITHUB_OUTPUT
```
but I've not tested it....